boards/stm32f412gdiscovery/src/main.rs

```diff
@@ -664,10 +664,12 @@ pub unsafe fn main() {
             // dc pin (optional)
             None,
             // reset pin
-            base_peripherals
-                .gpio_ports
-                .get_pin(stm32f412g::gpio::PinId::PD11)
-                .unwrap()
+            Some(
+                base_peripherals
+                    .gpio_ports
+                    .get_pin(stm32f412g::gpio::PinId::PD11)
+                    .unwrap()
+            )
```

Comment:
```suggestion
            base_peripherals
                .gpio_ports
                .get_pin(stm32f412g::gpio::PinId::PD11)
```
unless you still want to panic if `get_pin` fails

Reply:
I did not even think of this, thank you. :))